src/_guides/language/language-tour.md

```diff
@@ -1096,8 +1097,59 @@ var loudify = (msg) => '!!! ${msg.toUpperCase()} !!!';
 assert(loudify('hello') == '!!! HELLO !!!');
 {% endprettify %}
 
+This example uses an anonymous function.
+More about those in the next section.
 
-### Lexical scope {#lexical-scope}
+### Anonymous functions
+
+Most functions are named, such as `main()` or `printElement()`.
+You can also create a nameless function,
+called an _anonymous function_.
+An anonymous function is typically invoked once, or only a few times.
+You might assign an anonymous function to a variable so that,
+for example, you can add or remove it from a collection.
+
+An anonymous function looks similar to a named function&mdash;an optional parameter
```

Comment:
Not sure if you want to go into it here or not, but it can be multiple parameters.  Also the parameters can have types.